didx-xyz/yoma-mobile
# YOMA-575 - refactor(General): Extracting views from credential modules

## Scope [YOMA-XXX](https://yomaworld.atlassian.net/browse/YOMA-XXX)

![](https://media.giphy.com/media/l0MYOw8BXSIoeXVFC/giphy.gif)
## Work Done
- extracted all the views (incl widgets, form and individual views) into view-based modules that reflect the naming conventions in the designs.
- kept the user credential modules to specifically handle the data and state-matching. (so UserJobs, etc) 

## Steps to Test
- run all tests and ensure it's working
- see that MyCv view still works, all data is there, etc.
